modules/virt-about-vtpm-devices.adoc

```diff
@@ -18,6 +18,16 @@ the node has one.
 
 A vTPM device also protects virtual machines by storing secrets without physical hardware. {VirtProductName} supports persisting vTPM device state by using Persistent Volume Claims (PVCs) for VMs. You must specify the storage class to be used by the PVC by setting the `vmStateStorageClass` attribute in the `HyperConverged` custom resource (CR):
```

Comment:
I would change the colon at the end of this paragraph to a period, since you're adding more info in between this para and the code block.